src/app/modules/form/admin-form/__tests__/admin-form.routes.spec.ts

```diff
@@ -4929,7 +4937,43 @@ describe('admin-form.routes', () => {
         fields: expect.objectContaining({
           'Content-MD5': DEFAULT_POST_PARAMS.fileMd5Hash,
           'Content-Type': DEFAULT_POST_PARAMS.fileType,
-          key: DEFAULT_POST_PARAMS.fileId,
+          key: expect.any(String),
+          // Should have correct permissions.
+          acl: 'public-read',
+          bucket: expect.any(String),
+        }),
+      })
+      expect(response.body.fields.key).toEqual(
+        expect.stringContaining(DEFAULT_POST_PARAMS.fileId),
+      )
+      expect(DEFAULT_POST_PARAMS.fileId.length).toBeLessThan(
+        response.body.fields.key.length,
+      )
```

Comment:
any way to make this stricter? maybe do a regex check on the first 24 chars for the ObjectId

Reply:
thanks! done that